src/core/layout/qgslayoutpoint.cpp

```diff
@@ -26,7 +26,12 @@ QgsLayoutPoint::QgsLayoutPoint( const double x, const double y, const Qgis::Layo
   , mY( y )
   , mUnits( units )
 {
-
+#ifdef QGISDEBUG
+  if ( std::isnan( mX ) || std::isnan( mY ) )
+  {
+    qWarning( "Layout point with NaN coordinates created" );
```

Comment:
We don't want to Q_ASSERT on these scenarios to make it extra visible to devs?

Reply:
Ok, done!